### PR TITLE
fix: Add `database-name` parameter to RDS module & VPC CIDR block

### PR DESCRIPTION
**Pull request by LeonLuttenberger**
By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

**Comment by malachi-constant** [comment]
### AWS CodeBuild CI Report

* CodeBuild project: IDFModuleTestsInfrastructur-59JoFoVH64CI
* Commit ID: 0c54a88e0dbe95fe5f8f5fb43d50943c476d6a31
* Result: SUCCEEDED
* [Build Logs](https://mz6i2qapub.execute-api.us-west-2.amazonaws.com/Prod/buildlogs?key=58f18f3f-77a9-46ac-8610-3e8903a9376d%2Fbuild.log) (available for 30 days)

*Powered by [github-codebuild-logs](https://serverlessrepo.aws.amazon.com/applications/arn:aws:serverlessrepo:us-east-1:277187709615:applications~github-codebuild-logs), available on the [AWS Serverless Application Repository](https://aws.amazon.com/serverless/serverlessrepo/)*

File: manifests/local/database-modules.yaml [hunk 1]
```diff
@@ -61,6 +63,8 @@ parameters:
         group: networking
         name: basic-networking
         key: PrivateSubnetIds
+  - name: database-name
```
Comment (srinivasreddych) [inline manifests/local/database-modules.yaml:66]: repeated block of parameter

**Comment by srinivasreddych** [comment]
CHANGELOG is missing

File: manifests/local/database-modules.yaml (hunk 1 above)
Comment (LeonLuttenberger) [reply manifests/local/database-modules.yaml:66]: I define it once for the `rds` module and once for the `postgres` module. Each module needs it defined once.

**Comment by malachi-constant** [comment]
### AWS CodeBuild CI Report

* CodeBuild project: IDFModuleTestsInfrastructur-59JoFoVH64CI
* Commit ID: c12bd55fdcd3135e723e9c433d3288c09ca1c9e7
* Result: SUCCEEDED
* [Build Logs](https://mz6i2qapub.execute-api.us-west-2.amazonaws.com/Prod/buildlogs?key=847c79b7-3895-4fdd-8db6-a4032234a590%2Fbuild.log) (available for 30 days)

*Powered by [github-codebuild-logs](https://serverlessrepo.aws.amazon.com/applications/arn:aws:serverlessrepo:us-east-1:277187709615:applications~github-codebuild-logs), available on the [AWS Serverless Application Repository](https://aws.amazon.com/serverless/serverlessrepo/)*

**Comment by malachi-constant** [comment]
### AWS CodeBuild CI Report

* CodeBuild project: IDFModuleTestsInfrastructur-59JoFoVH64CI
* Commit ID: 777f6b00c1a7ab6f8e94c31a716bce0215ea744f
* Result: SUCCEEDED
* [Build Logs](https://mz6i2qapub.execute-api.us-west-2.amazonaws.com/Prod/buildlogs?key=ec4f373f-1fc1-4e17-a965-ba872d7ab25e%2Fbuild.log) (available for 30 days)

*Powered by [github-codebuild-logs](https://serverlessrepo.aws.amazon.com/applications/arn:aws:serverlessrepo:us-east-1:277187709615:applications~github-codebuild-logs), available on the [AWS Serverless Application Repository](https://aws.amazon.com/serverless/serverlessrepo/)*

**Review by malachi-constant — APPROVED** [review]

**Review by kukushking — APPROVED** [review]

**Pull request merged**